src/jbi/whiteboard_actions/default.py

```diff
@@ -5,36 +5,141 @@
+            )
+
+    def event_target_comment(self, payload: BugzillaWebhookRequest, linked_issue_key):
+        """Confirm issue is already linked, then apply comments; otherwise noop"""
+        if not linked_issue_key:
+            # noop
+            return JSONResponse(content={"status": "noop"}, status_code=201)
+        # else
+        jira_response = self.jira_client.issue_add_comment(
+            issue_key=linked_issue_key,
+            comment=payload.map_as_jira_comment(),
+        )
+        return JSONResponse(
+            content={"status": "comment", "jira_response": jira_response},
+            status_code=201,
+        )
+
+    def event_target_bug(  # pylint: disable=too-many-locals
+        self, payload: BugzillaWebhookRequest, linked_issue_key, bug_obj
+    ):
+        """Create and link jira issue with bug, or update; rollback if multiple events fire"""
+        if linked_issue_key:
+            # update
+            fields, comments = payload.map_as_tuple_of_field_dict_and_comments()
+            jira_update_response = self.jira_client.update_issue_field(
+                key=linked_issue_key, fields=fields
+            )
+            # comment
+            jira_comment_responses = []
+            for comment in comments:
+                jira_comment_responses.append(
+                    self.jira_client.issue_add_comment(
+                        issue_key=linked_issue_key, comment=comment
+                    )
+                )
+            return JSONResponse(
+                content={
+                    "status": "update",
+                    "jira_update_response": jira_update_response,
+                    "jira_comment_responses": jira_comment_responses,
+                },
+                status_code=201,
+            )
+        # else: create jira issue
+        fields = bug_obj.get_jira_issue_dict()
+        fields["project"] = {"key": self.jira_project_key}
+
+        response = self.jira_client.create_issue(fields=fields)
+
+        if isinstance(response, list):
+            response = response[0]
+
+        if isinstance(response, dict):
+            if any(
+                element in ["errors", "errorMessages"] and response[element]
+                for element in response.keys()
+            ):
+                # Failure to create: err keys exist with value
+                raise ActionError(f"response contains error: {response}")
+
+        jira_key_in_response = response.get("key")
+        if not payload.bug:
+            raise ActionError("payload does not contain bug data")
+        current_bug_info = self.bugzilla_client.getbug(payload.bug.id)
+        bug_obj = BugzillaBug.parse_obj(current_bug_info.__dict__)
+        jira_key_in_bugzilla = bug_obj.extract_from_see_also()
+        _duplicate_creation_event = (
+            jira_key_in_bugzilla is not None
+            and jira_key_in_response != jira_key_in_bugzilla
+        )
+        if not _duplicate_creation_event:
```

Comment:
Since the block in the case `_duplicate_creation_event` is shorter than the block `not _duplicate_creation_event`, I would invert it here. This way the long block below won't be indented and understood as the main code path

Reply:
Updated!